test/blackbox-tests/test-cases/dune-init/run.t

```diff
@@ -37,14 +37,14 @@ Clean up library with specified public name
 
 Can add a library with inline tests
 
-  $ dune init lib test_lib ./_inline_tests_lib --inline-tests --ppx ppx_inline_tests
+  $ dune init lib test_lib ./_inline_tests_lib --inline-tests --ppx ppx_inline_test
```

Comment:
I'm a bit confused as to how this command still work? Didn't you rename it to `library` in this PR?

Reply:
This works because Cmdliner's `enum` argument converter works based on the "unambiguous prefixes of string names" (see https://erratique.ch/software/cmdliner/doc/Cmdliner.Arg.html#converters). As a result, this PR doesn't *change* the names of the positional arguments (it is fully backwards compatible), it only improves the self-documentation of the code by fully spelling out the arguments. I still expect (and intend) `dune init (lib|test|exec|proj)` to be the way the command is invoked.

That said, I don't feel strongly about this change to the names, so if you'd rather I undo it, that's totally fine with me. Let me know whichever you'd prefer.

Reply:
Oh ok, thanks for the clarification. As long as it's backwards compatible, these names are fine by me.